pod/perldelta.pod

```diff
@@ -338,51 +935,500 @@ XXX
+
+=item *
+
+C<builtin::trim()> didn't properly clear C<TARG> which could result in
+out of date cached numeric versions of the value being used on a
+second evaluation.  Properly clear any cached values. [GH #22784]
+
+=item *
+
+L<perlfunc/shmread> and L<perlfunc/shmwrite> are no longer limited to 31-bit
+values for their POS and SIZE arguments.
+[L<GH #22895|https://github.com/Perl/perl5/issues/22895>]
+
+=item *
+
+L<perlfunc/shmread> is now better behaved if VAR is not a plain string.  If VAR
+is a tied variable, it calls C<STORE> once; previously, it would also call
+C<FETCH>, but without using the result.  If VAR is a reference, the referenced
+entity has its refcount properly decremented when VAR is turned into a string;
+previously, it would leak memory.
+[L<GH #22898|https://github.com/Perl/perl5/issues/22898>]
+
+=item *
+
+The C<$SIG{__DIE__}> and C<$SIG{__WARN__}> handlers can no longer be invoked
+recursively, either deliberately or by accident, as described in
+L<perlvar/%SIG>. That is, when an exception (or warning) triggers a call to a
+C<$SIG{__DIE__}> (or C<$SIG{__WARN__}>) handler, further exceptions (or
+warnings) are processed directly, ignoring C<%SIG> until the original
+C<$SIG{__DIE__}> (or C<$SIG{__WARN__}>) handler call returns.
+[L<GH #14527|https://github.com/Perl/perl5/issues/14527>], [L<GH #22984|https://github.com/Perl/perl5/issues/22984>], [L<GH #22987|https://github.com/Perl/perl5/issues/22987>]
+
+=item *
+
+The C<ObjectFIELDS()> for an object and C<xhv_class_fields> for the
+object's stash weren't always NULL or not-NULL, confusing C<sv_dump()>
+(and hence Devel::Peek's C<Dump()>) into crashing on an object with no
+defined fields in some cases.  [L<GH #22959|https://github.com/Perl/perl5/issues/22959>]
+
+=item *
+
+When comparing strings when using a UTF-8 locale, the behavior was
+previously undefined if either or both contained an above-Unicode code
+point, such as 0x110000.  Now all such code points will collate the same
+as the highest Unicode code point, U+10FFFF. [L<GH #22989|https://github.com/Perl/perl5/issues/22989>]
+
+=item *
 
-[ List each fix as an =item entry ]
+In regexes, the contents of C<\g{...}> backreferences are now properly
+validated. Previously, C<\g{1 FOO}> was silently parsed as C<\g{1}>, ignoring
+everything after the first number.
+[L<GH #23050|https://github.com/Perl/perl5/issues/23050>]
+
+=item *
+
+A run-time pattern which contained a code block which recursed back to the
+same bit of code which ran that match, could cause a crash.
+[L<GH #22869|https://github.com/Perl/perl5/issues/22869>]
+
+For example:
+
+    my $r = qr/... (?{ foo() if ... }) .../;
+    sub foo { $string =~ $r }
+    foo()
+
+=item *
+
+In some cases an C<eval> would not add integer parts to the source
+lines saved by the debugger. [L<GH #23151|https://github.com/Perl/perl5/issues/23151>]
+
+=item *
+
+Save debugger lines as C<PVIV> SVs rather than as C<PVMG> SVs as they
```

Comment:
This needs expansion to make sense.  And probably less detail

Reply:
And is this change actually visible

Reply:
I'm not capable of expanding that, though.

Reply:
Thank you a lot @khwilliamson and @mauke